.semaphore/multi-arch-builds-and-upload.yml

```diff
@@ -68,7 +78,7 @@ blocks:
       jobs:
         - name: "Build Native Executable (Linux AMD64)"
           commands:
-            - make mvn-package-native
+            - QUARKUS_NATIVE_RESOURCES_EXCLUDES="libs/**/*.dll,libs/**/*.dylib" make mvn-package-native
```

Comment:
Make sure we don't include native library files for Windows and macOS in the Linux executable.